lib/rubycritic/configuration.rb

```diff
@@ -29,7 +29,15 @@ def self.set(options = {})
     end
 
     def self.method_missing(method, *args, &block)
-      configuration.public_send(method, *args, &block)
+      if configuration
```

Comment:
It was a way to get the `super` into the `method_missing` to appease Rubocop.
It may just be better to disable the cop and put the code back how it was, but I wanted to at least present a version of the code that didn't raise any warnings.